core/internal/cltest/cltest.go

```diff
@@ -921,7 +924,15 @@ func WaitForPipeline(t testing.TB, nodeID int, jobID int32, expectedPipelineRuns
 			return true
 		}
 		return false
-	}, timeout, poll).Should(gomega.BeTrue(), fmt.Sprintf("job %d on node %d not %s with %d runs", jobID, nodeID, state, expectedPipelineRuns))
+	}, timeout, poll).Should(
+		gomega.BeTrue(),
+		fmt.Sprintf(`expected at least %d runs with status "%s" on node %d for job %d, got %d`,
+			expectedPipelineRuns,
+			state,
+			nodeID,
+			jobID,
+			actualRuns,
```

Comment:
Will `actualRuns` be evaluated for this message before the closure is actually executed to updated it?

Reply:
This should be no different to https://play.golang.org/p/KETzHVrqxxn